spec/requests/provider_interface/organisation_settings_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
+
+RSpec.describe 'Viewing organisation settings', type: :request do
+  let(:provider) { create(:provider, :with_signed_agreement) }
+  let(:provider_user) { create(:provider_user, providers: [provider], dfe_sign_in_uid: 'DFE_SIGN_IN_UID') }
+
+  before do
+    allow(DfESignInUser).to receive(:load_from_session)
+      .and_return(
+        DfESignInUser.new(
+          email_address: provider_user.email_address,
+          dfe_sign_in_uid: provider_user.dfe_sign_in_uid,
+          first_name: provider_user.first_name,
+          last_name: provider_user.last_name,
+        ),
+      )
+  end
+
+  def expect_redirect_to_account_page
+    expect(response.status).to eq(302)
+    expect(response.redirect_url).to eq(provider_interface_account_url)
+  end
+
+  describe 'GET index with feature flag off' do
+    it 'responds with 302' do
+      FeatureFlag.deactivate(:accredited_provider_setting_permissions)
```

Comment:
Could we extract the feature flag setting to a before block and use `context` to group the on/off feature flag behaviours?